README.md

```diff
@@ -210,17 +210,23 @@ You can run `eslint` on `.js`/`.jsx` files by doing:
 yarn lint-js
 ```
 
-## Transpiling
+## Transpiling the package and generating type definations for it
 
-You can transpile packages by doing:
+You can transpile a packages running the following commands in the root directory of the package:
 
 ```sh
-lerna run transpile
+yarn babel src -d dist --root-mode upward --extensions '.ts,.tsx' --ignore '**/*.test.ts,**/*.test.tsx,**/tests,**/__tests__'
```

Comment:
Can you include instructions for packages not written using Typescript?

Reply:
will do